bgpd/bgpd.c

```diff
@@ -3074,6 +3046,12 @@ int peer_group_remote_as(struct bgp *bgp, const char *group_name, as_t *as,
 	return 0;
 }
 
+void peer_notify_config_change(struct peer_connection *connection)
+{
+	if (BGP_IS_VALID_STATE_FOR_NOTIF(connection->status))
+		bgp_notify_send(connection, BGP_NOTIFY_CEASE, BGP_NOTIFY_CEASE_PEER_UNCONFIG);
```

Comment:
This should be BGP_NOTIFY_CEASE_CONFIG_CHANGE.

Reply:
yes sorry fixed

Reply:
forgot to push? :)